contribute/contribution-guidelines/pull-requests.md

```diff
@@ -120,14 +120,22 @@ If your code introduces deprecations, please note them here.
 
 ### Fixed ticket
 
-If your Pull Request resolves an existing issue, please note it using the _magic word_ "Fixes", followed by the issue number, like this: `Fixes #12314`.
+If your Pull Request resolves an existing issue, please note it using the _magic word_ "Fixes", followed by the issue number, like this: `Fixes #12314`. If your Pull Request fixes multiple issues, use following syntax: `Resolves #12314, Resolves #56789`.
```

Comment:
`Fixes` or `Resolves` ? 😄

Reply:
Fixes, but you must use `Resolves` word, if there are multiple issues :D

Reply:
You are right, i didn't knew this was specified 👍  (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)